ui/app/adapters/pki/issuer.js

```diff
@@ -21,6 +21,22 @@ export default class PkiIssuerAdapter extends ApplicationAdapter {
     }
   }
 
+  createRecord(store, type, snapshot) {
+    const { record, adapterOptions } = snapshot;
+    let url = this.urlForQuery(record.backend);
+    if (adapterOptions.import) {
+      url = `${url}/import/bundle`;
+    } else {
+      // TODO WIP generate
+      const type = 'root' || 'generate';
```

Comment:
The `type` here will never be `generate`, am I reading that right? If this is still WIP maybe it's best to throw an error or assertion here so we don't forget about it

Reply:
I think i mis-typed this and it should be root or intermediate